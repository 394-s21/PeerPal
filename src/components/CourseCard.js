import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import {Card, CardActionArea, CardActions, CardContent, Button, Typography} from '@material-ui/core';

const useStyles = makeStyles({
    root: {
      maxWidth: 275,
    },
    bullet: {
      display: 'inline-block',
      margin: '0 2px',
      transform: 'scale(0.8)',
    },
    title: {
      fontSize: 14,
    },
    pos: {
      marginBottom: 12,
    },
});

const CourseCard = ({name, id}) => {
    const classes = useStyles();

    return (
        <Card className = {classes.root}>
            <CardContent>
                <Typography className={classes.title}>
                    {name}
                </Typography>
                <Typography className={classes.pos}>
                    {id}
                </Typography>
            </CardContent>
        </Card>
     );
}
 
export default CourseCard;