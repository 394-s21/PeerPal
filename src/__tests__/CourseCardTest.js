
import renderer from 'react-test-renderer';
import CourseCard from '../components/CourseCard';
import {BrowserRouter as Router, Route, Switch} from 'react-router-dom'; 

test('Home Page Renders Correctly', () => {
  
  const name = "CS394"
  const id = 12345
  
  const page = renderer.create(
    <Router>
    <Route path="/course/:id">
    <CourseCard name={name} id={id}/>
    </Route>
    </Router>
    ).toJSON();
  expect(page).toMatchSnapshot();
});
