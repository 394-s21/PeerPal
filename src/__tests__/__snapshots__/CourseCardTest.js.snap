// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Renders Correctly 1`] = `null`;
